### DISCUSSION TITLE
eps_revisions always returns 'None' for 'downLast7days'

### DISCUSSION THREAD

**Discussion by avasfx** [ranaroussi/yfinance#2187] [General]
Hello. I updated yfinance to version 0.2.51. 
then:
`import yfinance as yf
tsla = yf.Ticker("TSLA")
tsla.eps_revisions`
and i get something like that:
<img width="507" alt="image" src="https://github.com/user-attachments/assets/b4a057db-1d85-4131-9158-970dadb481f6" />

But on yahoo finance site there is data for revisions:
<img width="1036" alt="image" src="https://github.com/user-attachments/assets/433c5e4a-5aaa-47a9-831e-4ab04542325b" />
The same is for other stocks. Am i missing something or yahoo changed something once again?

**Comment by ValueRaider** [comment]
Converted to issue #2188
